README.md

```diff
@@ -1,7 +1,7 @@
 # godot-rust-tutorials
 This is a tutorial project showing how to use the Rust Programming Language with Godot Game Engine
 
-Video materials associated with this tutorial project can be found under my youtube channel:
+Video materials associated with this tutorial project can be found under my youtube channel https://www.youtube.com/user/schr3da
```

Comment:
Probably needs to keep the `:`. eg:

`... youtube channel: https://www.youtube.com/user/schr3da`

:wink:

Reply:
Why keep the ':'? There is nothing behind in in README.md

Reply:
Just out of standard convention. :smile: